src/NzbDrone.Core/Organizer/FileNameBuilder.cs

```diff
@@ -315,74 +313,6 @@ private void AddQualityTokens(Dictionary<string, Func<TokenMatch, string>> token
             //tokenHandlers["{Quality Real}"] = m => qualityReal;
         }
 
-        private void AddMediaInfoTokens(Dictionary<string, Func<TokenMatch, string>> tokenHandlers, TrackFile trackFile)
-        {
-            if (trackFile.MediaInfo == null)
-            {
-                return;
-            }
-
-            var audioCodec = MediaInfoFormatter.FormatAudioCodec(trackFile.MediaInfo);
-            var audioChannels = MediaInfoFormatter.FormatAudioChannels(trackFile.MediaInfo);
-
-            var mediaInfoAudioLanguages = GetLanguagesToken(trackFile.MediaInfo.AudioLanguages);
-            if (!mediaInfoAudioLanguages.IsNullOrWhiteSpace())
-            {
-                mediaInfoAudioLanguages = $"[{mediaInfoAudioLanguages}]";
-            }
-
-            if (mediaInfoAudioLanguages == "[EN]")
-            {
-                mediaInfoAudioLanguages = string.Empty;
-            }
-
-            var mediaInfoSubtitleLanguages = GetLanguagesToken(trackFile.MediaInfo.Subtitles);
-            if (!mediaInfoSubtitleLanguages.IsNullOrWhiteSpace())
-            {
-                mediaInfoSubtitleLanguages = $"[{mediaInfoSubtitleLanguages}]";
-            }
-
-            var videoBitDepth = trackFile.MediaInfo.VideoBitDepth > 0 ? trackFile.MediaInfo.VideoBitDepth.ToString() : string.Empty;
-            var audioChannelsFormatted = audioChannels > 0 ?
-                                audioChannels.ToString("F1", CultureInfo.InvariantCulture) :
-                                string.Empty;
-
-            tokenHandlers["{MediaInfo Audio}"] = m => audioCodec;
-            tokenHandlers["{MediaInfo AudioCodec}"] = m => audioCodec;
-            tokenHandlers["{MediaInfo AudioChannels}"] = m => audioChannelsFormatted;
-
-            tokenHandlers["{MediaInfo Simple}"] = m => $"{audioCodec}";
-
-            tokenHandlers["{MediaInfo Full}"] = m => $"{audioCodec}{mediaInfoAudioLanguages} {mediaInfoSubtitleLanguages}";
```

Comment:
Should we keep the naming tokens in since we are still storing audio data, just from a different source?

Reply:
Can do.  Will simplify to codec and channels.  The tricky bit is dealing with old MediaInfo entries in the database Vs new Taglib based ones.  There's no mechanism for updating them any more unless the track is actually reimported.  Or I could do a DB migration...